### PR TITLE
Improve cli experience

### PR DESCRIPTION
**Pull request by matmair**
This PR:
- Makes sure inventree run cli works without activation
- Adds a version task to the invoke tasks (set as a default)

Printout of the version task
```

InvenTree - inventree.org
The Open-Source Inventory Management System

========================================
Installation paths:
Base        /opt/inventree
Config      /etc/inventree/config.yaml
Media       /opt/inventree/data/media
Static      /opt/inventree/data/static

Versions:
Python      3.8.10
Django      3.2.18
InvenTree   0.11.0
API         107

Commit hash:None
Commit date:None

You are probably running the package installer / single-line installer. Please mentioned that in any bug reports!

Use '--list' for a list of available commands
Use '--help' for help on a specific command
```

### CODE REVIEW

**Review by SchrodingersGat — APPROVED** [review]

**Comment by SchrodingersGat** [comment]
Nice addition :) ready to merge?

File: InvenTree/InvenTree/version.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 import django
 
-from InvenTree.api_version import INVENTREE_API_VERSION
+from .api_version import INVENTREE_API_VERSION
```
Comment (wolflu05) [inline InvenTree/InvenTree/version.py:12]: Why do you decided to import like this? What is the preferred way for imports? Sometimes I see you using relative imports. Sometimes you only  import the direct function, sometimes you import the complete module.

**Comment by matmair** [comment]
@SchrodingersGat yes!

File: InvenTree/InvenTree/version.py (hunk 1 above)
Comment (matmair) [reply InvenTree/InvenTree/version.py:12]: Imports in Python are a whole thing. In this specific case, the file will be called both by invoke from the context of / and from other functions in the Django app from the context of /InvenTree. So the absolute import would need to be both InvenTree.InvenTree.api_version and InvenTree.api_version.

We could set the working directory when starting invoke but that will lead to problems once external automation is used (ansible eg.). Therefore I decided that a relative import is the best solution here. In general, I prefer absolute imports for code that should be called from other modules and relative imports when a module is self-reliant and has a defined import point.

Comment (wolflu05) [reply InvenTree/InvenTree/version.py:12]: And what about the second half of my question?

> Sometimes you only import the direct function, sometimes you import the complete module.

Comment (matmair) [reply InvenTree/InvenTree/version.py:12]: It is easier to not import 20 different functions and there are some interesting corner cases with Django where it helps to import the module and not a specific function. Over 90% of the time it is the first reason

**Pull request merged**